swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -93,17 +93,11 @@ public final class Store: ObservableObject {
     #endif
   }
 
-  func createVPNProfile() {
-    Task {
-      try await TunnelManager.shared.create()
-
-      DispatchQueue.main.async { [weak self] in
```

Comment:
loadTunnelManager already wraps the UI updates in a DispatchQueue.main.async block. This was completely redundant.